lib/game_checker.rb

```diff
@@ -0,0 +1,23 @@
+class GameChecker
+  def winner?(board, player)
+    horizontal_winner?(board, player) || vertical_winner?(board, player) || diagonal_winner?(board, player)
+  end
+
```

Comment:
Why do you need to pass in the player if the cells have the marks?  Aren't you checking to see if all marks in the row, diagonal, or column are the same?